stock_quant_package_number/models/stock_quant_package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Camptocamp SA
+# @author: Italo LOPES <italo.lopes@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    package_number = fields.Char(compute="_compute_package_number", store=True)
+    order_number = fields.Char()
+    sheet_number = fields.Integer()
```

Comment:
I find the term not ideal to understand what it is
```suggestion
    parcel_number = fields.Integer()
```